Sprint-3/reading-list/style.css

```diff
@@ -157,3 +157,21 @@ body {
     max-height: 80px;
   }
 }
+
+#firstBookDiv {
```

Comment:
this wont work as its called bookDiv not firstBookDiv

Reply:
thank you so much for reviewing my codes.
I changed firstBookDiv to bookDiv, but got something unexpected so left it as before.